wiki/Tournaments/ICC/2021/en.md

```diff
@@ -0,0 +1,277 @@
+---
+tags:
+  - ICC 2021
+  - ICC2
+---
+
+# Indonesia Convert Cup 2021
+
+![ICC 2021 banner](img/banner.jpg)
+
+The **Indonesia Convert Cup 2021** (***ICC 2021***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [Chroneko](https://osu.ppy.sh/users/5472877) and  ![][flag_ID] [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in where all the mappools are exclusively made of converted osu! beatmaps. The tournament is open to all players from Indonesia with a playcount of 3,000 or above in osu!catch regardless of rank. It is the second iteration of the Indonesia Convert Cup.
```

Comment:
```suggestion
The **Indonesia Convert Cup 2021** (***ICC 2021***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [Chroneko](https://osu.ppy.sh/users/5472877) and ![][flag_ID] [Urabe Mikoto](https://osu.ppy.sh/users/1272422), where all mappools are exclusively made from converted osu! beatmaps. The tournament is open to all players from Indonesia with a playcount of 3,000 or above in osu!catch regardless of rank. It is the second iteration of the Indonesia Convert Cup.
```

Reply:
removed the extra space (whoops)

not sure about this one tho, i prefer my current sentence better personally :(